### DISCUSSION TITLE
How can I override the seek time when the user presses the left or right arrow keys?

### DISCUSSION THREAD

**Discussion by farhadibehnamdev** [vidstack/player#1068] [Q&A] [answered]
I am currently working on a project using Next.js v14.0.4, and I have integrated the Vidstack audio player. By default, the seek time when a user uses arrow keys is set at +5 and -5. However, I need to modify this functionality to make the seek time move by only 1 second instead.
@mihar-22 
```
 <MediaPlayer
            keyShortcuts={{
              seekBackward: ["ArrowLeft"],
              seekForward: ["ArrowRight"],
            }}
            title="Sprite Fight"
            src="/allears.mp3"
            crossorigin
            viewType="audio"
            ref={player}
            preload="metadata"
            onSeeked={onSeekedHandler}
          >
            <MediaProvider>
              <Track {...textTrack} />
            </MediaProvider>
            <AudioLayout />
          </MediaPlayer>
```

**Comment by mihar-22** [comment]
It can't be changed on the Default Layout at the moment. We can look at exposing it. Your other option is to look at building a custom layout and configuring the `TimeSlider` as desired.

**Comment by farhadibehnamdev** [comment]
Could you please provide an example of how to create a custom layout for configuring TimeSlider?
@mihar-22

**Reply by mihar-22** [reply, marked as answer]
Go through the docs and check out our [examples](https://github.com/vidstack/examples). You don't need to ping me at the end of each message btw :)
